<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="IN-PG0351S8" failures="1" tests="6" name="com.nagarro.cucumber.runners.TestRunner" time="76.731" errors="0" timestamp="2024-11-13T20:09:03 IST" skipped="0">
  <testcase classname="com.nagarro.cucumber.runners.TestRunner" name="runScenario" time="11.283"/>
  <testcase classname="com.nagarro.cucumber.runners.TestRunner" name="runScenario" time="5.158"/>
  <testcase classname="com.nagarro.cucumber.runners.TestRunner" name="runScenario" time="15.278"/>
  <testcase classname="com.nagarro.cucumber.runners.TestRunner" name="runScenario" time="7.778"/>
  <testcase classname="com.nagarro.cucumber.runners.TestRunner" name="runScenario" time="9.888"/>
  <testcase classname="com.nagarro.cucumber.runners.TestRunner" name="runScenario" time="27.346">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at com.nagarro.cucumber.stepdefinitions.LoginStepDefinition.an_error_message_should_be_displayed(LoginStepDefinition.java:76)
at âœ½.an error message should be displayed(file:///C:/Users/ashishyagyasaini/eclipse-workspace-1/Cucumber_Assignment/src/test/resources/features/login.feature:24)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.nagarro.cucumber.runners.TestRunner -->
